scripts/upgrade

```diff
@@ -17,6 +17,12 @@ sudo yunohost app upgrade $app -u https://github.com/YunoHost-Apps/gitea_ynh/com
     ynh_die "$message" --ret_code=1
 fi
 
+# If indexer doesn't exist, create it
+if [ -z "${indexer:-}" ]; then
+	indexer=0
```

Comment:
Using a longer var name, by example `repos_indexer` would be good to avoid any collision in long term.